tests/driver_soft_spi/Makefile

```diff
@@ -0,0 +1,14 @@
+APPLICATION = driver_soft_spi
+include ../Makefile.tests_common
+
+BOARD ?= stm32f4discovery
+
+USEMODULE += soft_spi
+
+CFLAGS += "-D SOFT_SPI_PARAMS_CUSTOM = {\
+                                           .miso_pin = GPIO_UNDEF,\
+																				   .mosi_pin = (GPIO_PIN(0, 0)),\
```

Comment:
Remove all these blank lines

Reply:
ah, you may have bigger tab space than i do